modules/ip6/control/route.c

```
@@ -30,6 +30,8 @@
 #include <string.h>
 #include <sys/queue.h>
 
+GR_LOG_TYPE("route");
```

Cppcheck (2.20.0)

[error] Uninitialized variable

(uninitvar)